test/popup.spec.js

```diff
@@ -284,52 +276,29 @@ module.exports.describe = function({testRunner, expect, playwright, CHROMIUM, WE
         const win = window.open('about:blank');
         win.alert('hello');
       });
-      const popupEvent = await page.waitForEvent('popup');
-      const popup = await popupEvent.page();
+      const popup = await page.waitForEvent('popup');
       const dialog = await popup.waitForEvent('dialog');
       expect(dialog.message()).toBe('hello');
       await dialog.dismiss();
       await evaluatePromise;
       await context.close();
     });
-    it.fail(FFOX)('should work with empty url', async({browser}) => {
+    it('should work with empty url', async({browser}) => {
```

Comment:
Let's keep a test which opens an empty url, and then calls `waitForLoadState({waitUntil:'load'})`.

Reply:
Done, added to Page.waitForLoadState suite